hack/vendor.sh

```diff
@@ -18,7 +18,7 @@ clone git github.com/microsoft/hcsshim 7f646aa6b26bcf90caee91e93cde4a80d0d8a83e
 clone git github.com/mistifyio/go-zfs v2.1.1
 clone git github.com/tchap/go-patricia v2.1.0
 clone git golang.org/x/net 3cffabab72adf04f8e3b01c5baf775361837b5fe https://github.com/golang/net.git
-clone hg code.google.com/p/gosqlite 74691fb6f837
+clone git github.com/mxk/go-sqlite 167da9432e1f4602e95ea67b67051cfa34412e3f
```

Comment:
As far as _time_ is concerned, https://github.com/mxk/go-sqlite/commit/1ad6fca94c61dc5cb4862c3b645653b2f58bc7fc is pretty close to what we were using before. (May 30, 2013 vs May 25, 2013; https://code.google.com/p/gosqlite/source/detail?r=74691fb6f83716190870cde1b658538dd4b18eb0)

Not sure if that actually means anything, and haven't had a chance to check the code diff yet.